src/main/scala/edu/colorado/plv/cuanto/jsy/binding/Parser.scala

```diff
@@ -0,0 +1,77 @@
+package edu.colorado.plv.cuanto.jsy
+package binding
+
+import edu.colorado.plv.cuanto.jsy.common.{JsyParserLike, OpParserLike, UnitOpParser}
+
+import scala.util.parsing.input.Positional
+
+/** Parse variables and bindings.
+  *
+  * @author Bor-Yuh Evan Chang
+  */
+trait ParserLike extends OpParserLike with JsyParserLike {
+  import ParserLike._
+
+  override def stmt: Parser[Expr] = {
+    rep(concreteStmt) ^^ { stmts =>
+      stmts.foldRight(None: Option[Expr]) {
```

Comment:
As I think I said in #17, I am a bit torn about this, as the alternative would be to treat \n as a token. The statement-expression layering helps some, but it does allow some of these statement blocks that seem a bit odd.